jwcrypto/jwt.py

```diff
@@ -418,6 +457,16 @@ def deserialize(self, jwt, key=None):
         # have a key
         self.token.deserialize(jwt, key)
 
+        if keyset is not None:
```

Comment:
we should probably test that keyset is an actual JWKset and raie a proper error if not ?